### PR TITLE
KeystoneService API update

### PR DESCRIPTION
**Pull request by slagle**

### CODE REVIEW

File: config/samples/keystone_v1beta1_keystoneservice.yaml [hunk 1]
```diff
@@ -3,5 +3,18 @@ kind: KeystoneService
 metadata:
   name: keystoneservice-sample
 spec:
-  # Add fields here
-  foo: bar
+  username: admin
```
Comment (dprince) [inline config/samples/keystone_v1beta1_keystoneservice.yaml:6]: Should we have this input be a secret?

File: config/samples/keystone_v1beta1_keystoneservice.yaml [hunk 2]
```diff
@@ -3,5 +3,18 @@ kind: KeystoneService
...
+  password: foobar123
```
Comment (dprince) [inline config/samples/keystone_v1beta1_keystoneservice.yaml:7]: Should we have this input for 'password' use a k8s secret?

Comment (mpryc) [reply config/samples/keystone_v1beta1_keystoneservice.yaml:7]: Note the CI failures:
https://travis-ci.com/github/openstack-k8s-operators/keystone-operator/jobs/384834414

**Review by dprince — APPROVED** [review]

**Comment by dprince** [comment]
@slagle looks good to me. Haven't tested it yet but if it works I'd say merge when ready. Will be nice to be able to use this across the board

**Pull request merged**